tests/test_enum.rs

```diff
@@ -152,3 +152,27 @@ fn test_id_pat_with_id() {
     );
     assert_eq!(input, &*v.to_bytes().unwrap());
 }
+
+#[derive(PartialEq, Debug, DekuRead, DekuWrite)]
+#[deku(type = "u16", id_endian = "big", endian = "little")]
```

Comment:
This needs updated for the latest changes.

```suggestion
#[deku(id_type = "u16", id_endian = "big", endian = "little")]
```